reservation/src/main/resources/properties/config.yml

```
@@ -27,4 +27,4 @@ spring:
   config.activate.on-profile: prod
   cloud:
     config:
-      enabled: true
\ No newline at end of file
+      enabled: false
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)